src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -163,10 +164,21 @@ type BidderSettings = {
 	magnite?: Partial<BidderSetting>;
 };
 
+const shouldIncludeGpid = isUserInVariant(gpidPrebidAdUnits, 'variant');
```

Comment:
This still won't rollout to 99% of users yet it needs to be the inverse of this. AKA shouldIncludeGpid = true if not in the control

Reply:
Ah yes thanks Jake